### DISCUSSION TITLE
Reloading Results in R2D

### DISCUSSION THREAD

**Discussion by mlochhead** [NHERI-SimCenter/SimCenterCommon#305] [R2D] [answered]
Once you have done an R2D analysis (a building analysis of a city using Hazus, for example), is it possible to reload the results to view them in R2D after the fact? Thanks!

**Comment by jinyan1214** [comment, marked as answer]
Thanks for your questions.

Yes, you can load the analysis results back to R2D. There is a "Results" button next to the "Tools" dropdown menu at the top bar of the user interface. Clicking the "Load Results" button under "Results" and input "inputRWhale.json" as "Input file" and the "Results" folder in "tmp.SimCenter" as "Results folder." This should allow you to re-load the results.
